kokoro/scripts/test/go_test.sh

```diff
@@ -40,13 +38,15 @@ source kokoro/scripts/utils/common.sh
 track_flakiness
 
 # If a built agent was passed in from Kokoro directly, use that. The file will
-# always be in KOKORO_GFILE_DIR, though it may not be in the same subdirectory,
-# so we have to search for it.
+# always be in $KOKORO_GFILE_DIR/result or $KOKORO_GFILE_DIR/out.
 if [[ -d "${KOKORO_GFILE_DIR}" ]]; then
-  BUILT_AGENT_PATH=$(find "${KOKORO_GFILE_DIR}" -type f -name "google-cloud-ops-agent*" | head -n 1)
-  if [[ ! -z "${BUILT_AGENT_PATH}" ]]; then
-    RESULT_DIR="$(dirname "${BUILT_AGENT_PATH}")"
-
+  if compgen -G "${KOKORO_GFILE_DIR}/result/google-cloud-ops-agent*" > /dev/null; then
+    RESULT_DIR="${KOKORO_GFILE_DIR}/result"
+  elif compgen -G "${KOKORO_GFILE_DIR}/out/google-cloud-ops-agent*" > /dev/null; then
+    RESULT_DIR="${KOKORO_GFILE_DIR}/out"
+  fi
+  
```

Comment:
Looks like there may be some trailing whitespace here…

Reply:
fixed